ci/runJob.groovy

```diff
@@ -0,0 +1,395 @@
+def gfxrTestWindows(

```

Comment:
This is acceptable, though I'm not thrilled with the duplication of code between the `gfxrTestWindows()`, `gfxrTestLinux()`, and `gfxrTestAndroid()` functions (and with the implicit lack of `gfxrTestMac()` - perhaps `gfxrTestPosix()` might be better?).

These functions all function identically, except for the use of `bat` or `sh`, and the use of `runjob.bat` or `runjob.sh`, yes?  Is there some way to parametrize this so the common code can be factored out?